storage/redis_store.go

```diff
@@ -150,6 +150,11 @@ func NewRedisStore(address string, isCluster bool, db int, timeout time.Duration
     var tlsConfig *tls.Config
     var client redis.Cmdable
 
+    if os.Getenv("REDIS_TLS") != "" {
```

Comment:
this is already in `main`, I believe?
can you please do:
1. update your `main` branch to be in sync with this repo's `main`
2. rebase your branch `add-redis-cluster` on your `main` and resolve conflicts (if any)
3. make sure you run tests (`make test`) and they all pass
thanks.

Reply:
yep, see [here](https://github.com/massenz/go-statemachine/blob/main/storage/redis_store.go#L151) it's been merged in `main` and `release` and it's in Release `0.6.0`